diff --git a/guidance-power-manager.py b/guidance-power-manager.py
index 70a2bdb..229449d 100755
--- a/guidance-power-manager.py
+++ b/guidance-power-manager.py
@@ -160,7 +160,7 @@ class PowerManager(QWidget):
     def _checkOldConfig(self, value, blank):
         """ Convert old numerical values to keywords. """
         try:
-            num_val = int(value)
+            num_val = value.toInt()[0]
         except ValueError:
             return value
         if blank:
